src/components/auth/Register.js

```diff
@@ -6,7 +6,7 @@ export const Register = (props) => {
     const firstName = useRef()
     const lastName = useRef()
     const email = useRef()
-    const bio = useRef()
+    const displayName = useRef()
```

Comment:
can this be display_name so all the variables are the same as what will be in the database